DestructionProcessing.cs

```diff
@@ -471,6 +471,8 @@ internal static void HandleVehicleCap()
                 {
                     if (vehicle.isDead)
                         continue;
+                    if (vehicle.passengers != null)
```

Comment:
The first section if to build the list, the second section is to process through the list. While the dead vehicles can be safely ignored in the list building section, it would be better to not have the passengers check in the list building section, so the vehicle count to start the vehicle processing wouldn't be affected when there's players in the cars on the server. Leaving the second check would have that vehicle skipped over when the list is processed.